src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -9,7 +9,6 @@
     <TargetRefPath>$(OutputPath)$(TargetFramework)\ref\$(MSBuildProjectName).dll</TargetRefPath>
     <IsPackable>true</IsPackable>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <GeneratePlatformNotSupportedAssembly Condition="'$(RuntimeIdentifier)' == ''">true</GeneratePlatformNotSupportedAssembly>
```

Comment:
We can't remove this. This needs to be in the package in case it is restored in a platform where we haven't provided any implementation for, for example, on OSX. There is currently a plan we have to remove the Windows and Linux specific builds and instead just keep one implementation which works in both places, which would make it so that we can remove this guy, but for now we need to keep it.